src/main.java

```diff
@@ -7,23 +7,36 @@ public class main {
 
 	LinkedList<String> listOfFiles = new LinkedList<String>();
 
+	String Path = "C:\\Users\\Gemst\\OneDrive\\autosync\\Dokument\\GitHub\\weatherGenerator\\";
+	
 	public void run(String[] args, String Path, String file, int startMonth){
 		for(int i = 0; i<args.length; i++)
 		{
 			if(args[i] == "-path") {
```

Comment:
släng in en guard för OOB
```suggestion
			if(args[i] == "-path" && (i+1)<= args.length) {
```